CMakeLists.txt

```diff
@@ -7,16 +7,28 @@
 # Version 3.15 is the baseline for P4 Control Plane.
 cmake_minimum_required(VERSION 3.15)
 
-project(krnlmon LANGUAGES C CXX)
+project(krnlmon VERSION 2.0 LANGUAGES C CXX)
```

Comment:
Are we (planning to) giving each submodule a version number?

Reply:
I'm leaning in favor of versioning cmake projects that are, or may be considered to be, discrete _packages_ or _components_.

krnlmon already had a version number of sorts, defined by the `VERSION` variable later in this file and substituted into the `libkrnlmon` pkg-config file.

This CL moves the definition to the cmake `project()` command, bumps the version number from 1.0 to 2.0, and replaces the `VERSION` variable with `PROJECT_VERSION` when creating the pkg-config file.

CMake defines `PROJECT_VERSION` when you specify `project(VERSION x.x)`.